README.md

```diff
@@ -0,0 +1,35 @@
+# terascope helm chart repository
+
+## Overview
+
+This repository holds helm charts useful for running services for [teraslice](https://github.com/terascope/teraslice).
+
+## Documentation
+
+- <https://terascope.github.io/helm-charts/>
+
+```bash
+helm repo add terascope https://terascope.github.io/helm-charts/
+helm repo update
+```
+
+## Development
+
+This repo requires the following tools to perform pre-commit hooks:
+
+- `pre-commit`
+  - see the install guide here: <https://pre-commit.com/#install>
```

Comment:
I don't know that I like the python requirement this imposes on the developers workstation.  Would [Husky](https://typicode.github.io/husky/) provide the same functionality?  It runs on node so the dev will already have the runtime installed.

Reply:
Switching to husky works well. Developers can run a single command, `npx husky install`, to set up hooks in the repo without installing anything(assuming they have `npm/npx` already).

Reply:
Ok, make that change.